mqtt/mqtt-broker/src/auth.rs

```diff
@@ -29,14 +29,14 @@ mod imp {
     use std::error::Error as StdError;
 
     use mqtt_broker_core::auth::{
-        authenticate_fn_ok, authorize_fn_ok, AuthId, Authenticator, Authorizer,
+        authenticate_fn_ok, authorize_fn_ok, AuthId, Authenticator, Authorization, Authorizer,
```

Comment:
Oops. Second time in a row. We probably need to add another step to build/check with `--no-default-features` flag